docs/pages/ESSupport.md

```diff
@@ -167,6 +167,30 @@ A library is typically initialized via an accompanying `library.js`. Within that
     });
     ```
 
+#### Reserved Variable Names in a Template Literal
+
+Template Literals with an expression requires to relinquish following variable names:
```

Comment:
From the top of my head, I did not know the meaning of `relinquish` 😄 

How about:

```suggestion
Template Literals with an embedded expression must not use the following variable names:
```

Reply:
The reason for this limitation should be explained. We could even suggest to use https://eslint.org/docs/latest/rules/template-curly-spacing with option "always" to mitigate this. Our placeholder replacement does not allow spaces.

Reply:
Tackled both comments.